ui/app/services/custom-messages.js

```diff
@@ -29,14 +30,15 @@ export default class CustomMessageService extends Service {
 
   async fetchMessages(ns) {
     try {
-      const url = '/v1/sys/internal/ui/unauthenticated-messages';
+      const url = this.auth.currentToken
+        ? '/v1/sys/internal/ui/authenticated-messages'
+        : '/v1/sys/internal/ui/unauthenticated-messages';
       const opts = {
         method: 'GET',
         headers: {},
       };
-      if (ns) {
-        opts.headers['X-Vault-Namespace'] = ns;
-      }
+      if (this.auth.currentToken) opts.headers['X-Vault-Token'] = this.auth.currentToken;
```

Comment:
there's a chance this might become lower case (`x-vault-token`) after the ember data upgrade. If this fails, will a test catch it?

Reply:
There isn't currently a test to catch this. Do you have suggestions on how to test this? :)

Reply:
Will tackle this service test in a different PR